src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,621 @@
+                          {^Array: {of: {^Choose: {from: *TenArrFields, deterministic: 1}}, number: 50}}
+                       }}}]
+
+- ExpressionConcatArraysHundred:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionConcatArraysHundred
+      nopInPhasesUpTo: *MaxPhases
+      active: [11]
+      operationMetricsName: VariadicAggExpressionConcatArraysHundred
+      Pipeline: [{$project:
+                    {concatArrays:
+                       {$concatArrays:
+                          {^Array: {of: {^Choose: {from: *TenArrFields, deterministic: 1}}, number: 100}}
+                       }}}]
+
+# For the following $ifNull test cases, the arguments do not exist in the
+# documents that we insert into our collection. This results in executing
+# every argument to $ifNull
+- ExpressionIfNullTen:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNullTen
+      nopInPhasesUpTo: *MaxPhases
+      active: [12]
+      operationMetricsName: VariadicAggExpressionIfNullTen
+      Pipeline: [{$project:
+                    {ifNull:
+                       {$ifNull:
+                          {^Concat:
+                             {arrays:
+                                [{^Array: {of: {^Choose: {from: *TenNonExistentFields, deterministic: 1}}, number: 10}}, [0]]}}
+                       }}}]
+
+- ExpressionIfNullFifty:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNullFifty
+      nopInPhasesUpTo: *MaxPhases
+      active: [13]
+      operationMetricsName: VariadicAggExpressionIfNullFifty
+      Pipeline: [{$project:
+                    {ifNull:
+                       {$ifNull:
+                          {^Concat:
+                             {arrays:
+                                [{^Array: {of: {^Choose: {from: *TenNonExistentFields, deterministic: 1}}, number: 50}}, [0]]}}
+                       }}}]
+
+- ExpressionIfNullHundred:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNullHundred
+      nopInPhasesUpTo: *MaxPhases
+      active: [14]
+      operationMetricsName: VariadicAggExpressionIfNullHundred
+      Pipeline: [{$project:
+                    {ifNull:
+                       {$ifNull:
+                          {^Concat:
+                             {arrays:
+                                [{^Array: {of: {^Choose: {from: *TenNonExistentFields, deterministic: 1}}, number: 100}}, [0]]}}
+                       }}}]
+
+- ExpressionLetTen:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionLetTen
+      nopInPhasesUpTo: *MaxPhases
+      active: [15]
+      operationMetricsName: VariadicAggExpressionLetTen
+      Pipeline: [{$project:
+                    {let:
+                       {$let:
+                          {vars:
+                             {
+                               a: "$int1", b: "$int2", c: "$int3", d: "$int4", e: "$int5",
```

Comment:
Similar comment, can we automatically generate these fields instead of hard coding them? One idea is to bind the referenced fields by name, but then auto-generate random fields for the non referenced fields.

Reply:
Done.